shared/src/business/entities/externalDocument/ExternalDocumentStandard.test.js

```diff
@@ -0,0 +1,39 @@
+const { ExternalDocumentFactory } = require('./ExternalDocumentFactory');
+
+describe('ExternalDocumentStandard', () => {
+  describe('validation', () => {
+    it('should have error messages for missing fields', () => {
+      const extDoc = ExternalDocumentFactory.get({
+        scenario: 'Standard',
+      });
+      expect(extDoc.getFormattedValidationErrors()).toEqual({
+        category: 'You must select a category.',
+        documentType: 'You must select a document type.',
+      });
+    });
+
+    it('should be valid when all fields are present', () => {
+      const extDoc = ExternalDocumentFactory.get({
+        category: 'Application',
+        documentTitle: 'Application for Waiver of Filing Fee',
+        documentType: 'Application for Waiver of Filing Fee',
+        scenario: 'Standard',
+      });
+      expect(extDoc.getFormattedValidationErrors()).toEqual(null);
+    });
+  });
+
+  describe('title generation', () => {
+    it('should generate valid title', () => {
+      const extDoc = ExternalDocumentFactory.get({
+        category: 'Application',
+        documentTitle: 'Application for Waiver of Filing Fee',
+        documentType: 'Application for Waiver of Filing Fee',
+        scenario: 'Standard',
+      });
+      expect(extDoc.getDocumentTitle()).toEqual(
+        'Application for Waiver of Filing Fee',
```

Comment:
This test case is using an actual `Standard` document and checking what that the document title is what is expected. a `Standard` document has no placeholders that need to be replaced. So it would be the same title.

These items are going through a polymorphic factory, so all of the implementations would need to have the same functions.

Reply:
Ah, OK — I understand. Thank you.